### DISCUSSION TITLE
Open Source setup?

### DISCUSSION THREAD

**Discussion by coffseducation** [NightscoutFoundation/xDrip#3265] [Q&A]
Hello! Setup advice needed! 
I was looking to install Nightscout on a VPS with Caprover but I am confused with the amount of info for the scanning and data collection side. 

What is the best way to collect data so that myself and siblings can monitor our mothers data and receive warnings - while staying open source? My mothers phone runs open source lingeageOS that I gifted her. (She is not technical). As such the NFC function does not work - but I think I found an app to add functionality if neccessary. 
She uses a separate reader to scan the Freestyle Libre 2 sensor. 

What method should I use to collect data while allowing my elderly mother to continue to use the reader? She will not want to use a phone for scanning. Can data be transmitted to the phone from the sensor or reader without disrupting her current routine? 

nb. Open source for data monitoring and collection - but monitoring from fruit devices by the other siblings.

Thanks!

**Comment by Navid200** [comment]
This explains how you can set up Nightscout on a Google Cloud virtual machine:
https://navid200.github.io/xDrip/docs/Nightscout/GoogleCloud.html

On Android, xDrip can read from Nightscout.
On iOS, there are apps that can read from Nightscout.

As far as using Libre 2, there are people who use xDrip to read from Libre 2.
And xDrip can upload to Nightscout.

**Reply by coffseducation** [reply]
Hello Navid200!

Thanks for the reply!
I will be setting Nightscout up in Caprover with a docker install, so the google cloud info is not relevant for me.

My question really is how to setup xDrip to collect data - but still have the Libre stand alone reader do the collection - at least for my mother. Can xDrip be linked as a secondary collection point without having to use it to physically scan, or decommission the Libre Reader?

Thanks!

**Reply by Navid200** [reply]
@PhilTU

**Comment by PhilTU** [comment]
> My question really is how to setup xDrip to collect data - but still have the Libre stand alone reader do the collection - at least for my mother. Can xDrip be linked as a secondary collection point without having to use it to physically scan, or decommission the Libre Reader?
> 

the reader device does not transmit and can only be interrogated by USB so it does not offer a way to follow someone's data remotely.

The phone would require NFC in order to set up Bluetooth link to xDrip+ or other app, as that's how the security is done. Some other CGMs use QR codes and pure Bluetooth connections.

In principle you can use xDrip+ to "steal" the bluetooth data stream from the sensor so that xDrip+ receives a value every minute which can be used for alarms, sending to Nightscout etc. There are however two issues - your mother would lose alarms on the reader device, and said reader device would continually try to reconnect to the sensor - potentially disrupting the link to xDrip+

Some people say they can get along ok with the reader present, but I don't know if it's in a faraday bag / microwave over or at the other end of the house - generally these folks are monitoring via xDrip+ on the phone and only use the reader for collecting data for healthcare services a few times a day.

**Reply by coffseducation** [reply]
Thank you for the detailed response.
It would seem then I have to convince my mother to use the phone (provided I get NFC working)
or
abandon the idea of Nightscout.

Thanks!
